src/components/CompiledOutput.js

```diff
@@ -1,10 +1,18 @@
 import React, { useEffect, useState } from "react";
-import * as Babel from "@babel/core";
+import * as Babel from "@babel/standalone";
 import { processOptions } from "../standalone";
 import { gzipSize } from "../gzip";
 import { Wrapper, Code, Config } from "./styles";
 import { useDebounce } from "../utils/useDebounce";
 
+import {
+  convertToBabelConfig,
+  //   importDefaultPlugins,
+  //   registerDefaultPlugins
```

Comment:
Either remove these comments or add more details explaining what they're doing here.

Reply:
Oops, there's some commented snippets of code here and there that I forgot to remove.